wherehows-web/app/components/dataset-compliance.ts

```diff
@@ -296,27 +295,32 @@ export default class DatasetCompliance extends Component {
     };
 
     /**
-     * Inserts a divider in the list of compliance field identifier dropdown options
+     * Inserts a divider in the list of compliance field identifier options
      * @param {Array<IComplianceFieldIdentifierOption>} types
      * @returns {Array<IComplianceFieldIdentifierOption>}
      */
-    const insertDivider = (types: Array<IComplianceFieldIdentifierOption>): Array<IComplianceFieldIdentifierOption> => {
+    const insertDividers = (
+      types: Array<IComplianceFieldIdentifierOption>
+    ): Array<IComplianceFieldIdentifierOption> => {
       const isId = ({ isId }: IComplianceFieldIdentifierOption): boolean => isId;
       const ids = types.filter(isId);
       const nonIds = types.filter((type): boolean => !isId(type));
-      const divider = {
-        value: '',
-        label: '---------',
-        isId: false,
-        isDisabled: true
-      };
-
-      return [...ids, <IComplianceFieldIdentifierOption>divider, ...nonIds];
+      //divider to indicate section for ids
+      const idsDivider = { value: '', label: 'IDs', isDisabled: true };
+      // divider to indicate section for non ids
+      const nonIdsDivider = { value: '', label: 'Non IDs', isDisabled: true };
+
+      return [
+        <IComplianceFieldIdentifierOption>idsDivider,
```

Comment:
Why do we need to assert type here? Seems like the idsDivider should already fit the typing.